source/index.md

```diff
@@ -24,10 +24,18 @@ to 3.1 or 3.0 as soon as possible. Extended support for 1.0.2 to gain access to
 security fixes for that version is [available](/support/contracts.html).
 
 
-The following OpenSSL version(s) are FIPS validated.
+The following OpenSSL version(s) are FIPS validated:
```

Comment:
Should we say somewhere here that you should use the latest OpenSSL version for the libraries in conjunction with the validated version for the FIPS provider?

Reply:
This is mentioned just after the table:

> Other OpenSSL Releases MAY use the validated FIPS provider, but MUST NOT build and use their own FIPS provider. For example you can build OpenSSL 3.1 and use the OpenSSL 3.0.0 FIPS provider with it.

It's not as forceful as what you are suggesting.